site/_posts/2020-06-05-private-labeling-with-fusionauth.md

```diff
@@ -0,0 +1,107 @@
+
+You realize you can offer this easily by modifying your application to respond to multiple hostnames. The logic is pretty straightforward.
+
+But what about your users? The accounts at todo.company1.com should be entirely separate from the accounts at app.todo.com. What if the CEO of Company1 has an account at app.todo.com and signs up for a corporate account? It doesn't make sense to mix personal and business tasks. Further, suppose you sell Organization2 a premium subscription as well. These users should be separate too. 
+
+Also, each standalone application's login pages must be branded; your clients want their apps customized to match their websites. Oh, and by the way, Company1 wants users to authenticate against ActiveDirectory, and Organization2 wants users to be able to login with their Facebook accounts. 
+
+_What to do?_
+
+We got ya. You can handle this scenario in two different ways with FusionAuth.
+
+## Separate FusionAuth servers
+
+The first option is running separate FusionAuth instances on separate servers, each with their own backing database. This is easy to understand and has some strengths: 
+
+* You can scale each installation independently. 
+* The servers can be located in different legal jurisdictions. 
+* The FusionAuth admin UI can be made available to premium clients if desired.
+
+However, operationally this choice leads to complexity. There's the cost of running and maintaining the different servers. You'll need to make sure that configuration such as admin accounts, webhooks, and API keys are synced. When any employee of TodoInc departs, you'll need to ensure you remove accounts across all the servers. 
+
+You'll also need to automate your server rollout process so that todo.company1.com doesn't get left behind when you upgrade. And you'll need some way to let your customer service folks know which installation is associated with which client, so that when a request comes in to reset a password, they aren't hunting across different servers. This isn't a big issue when you have only three private labeled accounts, but when you have twenty or two hundred, it becomes problematic.
+
+## Tenants to the rescue
+
+FusionAuth provides an easier option: tenants. [Tenants](/docs/v1/tech/core-concepts/tenants) are a first class construct in FusionAuth. When you set up a new instance, there is one tenant installed: "Default". And sometimes one is enough. 
+
+But you can create as many as you'd like. From the perspective of user signing in, each tenant is a separate installation. Each tenant has its own email templates, themes, application configurations and users. API keys can be scoped to a tenant, so if you want to give a client an API key to allow them to create their own integrations, you can. This allows for tighter integrations between your todo application and your clients' systems, and is a nice premium feature to offer with no cost to you.
```

Comment:
"From the perspective of user signing in" should be "From the perspective of a user signing in".